### PR TITLE
integrated kyc in finance tab

### LINKED ISSUE #140: Redesign: /Finance/KYC
**NOTICE** 

**Remember to checkout and send a pull request to the [redesign](https://github.com/openMF/mobile-wallet/tree/redesign)  branch.**

---

We are working on the app redesign. The prototype is done and can be found here: [show prototype](https://xd.adobe.com/view/8581e34d-8b91-49ec-7c21-000d98c8f18b-f3b0/?hints=off), and source files here: [show repository](https://github.com/letelete/mifos-pay-app-design). We will be working on the redesign implementation now.

This issue is about implementing the design and features of the KYC tab in the Finance section.

That's what we want to achieve:

image

### PR DESCRIPTION
**Pull request by manddy**
Fixes 
#140 

Screenshots
ss

Summary
Integrated the kyc section in finance tab and changed the design as per requirement

### CODE REVIEW

**Comment by manddy** [comment]
@ankurs287 @luckyman20  kindly look at the PR

**Comment by luckyman20** [comment]
Shouldn't there be relation between all the 3 KYC Level Tabs?

**Comment by manddy** [comment]
> Shouldn't there be relation between all the 3 KYC Level Tabs?
The three tabs are independent of each other the user will go to the next level after completing the previous level which is implemented in the code .If there anything else to point out kindly tell me about it .

**Comment by luckyman20** [comment]
They are related, you can only apply for KYC Level 2 if you have completed Level 1.

**Comment by manddy** [comment]
> They are related, you can only apply for KYC Level 2 if you have completed Level 1.

@luckyman20  I have actually used the master branch code for kyc and in the master branch this condition is not satisfied as well.I can make a separate issue for it and implement this thing ,also can you just review this pull request and give me some review on it

**Comment by luckyman20** [comment]
@manddy If it is not there, you should do it in this PR only. As making a separate issue and then a separate PR would only delay this feature.

**Comment by manddy** [comment]
> @manddy If it is not there, you should do it in this PR only. As making a separate issue and then a separate PR would only delay this feature.

yes,I have  rectified this issue in my code

**Comment by manddy** [comment]
> @manddy If it is not there, you should do it in this PR only. As making a separate issue and then a separate PR would only delay this feature.

can you please tell me the variable  names i have to change ,I have already tried to name most of them correctly  but can you specifically point out which of them I have to rectify?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java [hunk 1]
```diff
@@ -37,23 +41,41 @@
 
     KYCContract.KYCDescriptionPresenter mKYCDescriptionPresenter;
 
-    @BindView(R.id.cv_lvl1)
-    CardView cvLvl1;
+    @BindView(R.id.lllv1)
+    LinearLayout lllv1;
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java]: Keep it `lvl1` only and change for other Linear Layouts too

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java [hunk 2]
```diff
@@ -37,23 +41,41 @@
...
+    @BindView(R.id.blv2)
+    Button blv2;
 
-    @BindView(R.id.ll_lvl3)
-    LinearLayout llLvl3;
+    @BindView(R.id.blv3)
+    Button blv3;
+
+    @BindView(R.id.completedtx1)
+    TextView txt1;
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java]: Keep these as same as the id. Like for this one you would change it to `completedtx1`.

**Comment by manddy** [comment]
@luckyman20  @ankurs287  Done all changes kindly check

**Comment by ankurs287** [comment]
@manddy good job

**Pull request merged**